web/src/SidebarResources.tsx

```diff
@@ -129,6 +123,65 @@ let SidebarItemTimeAgo = styled.span`
 type Resource = Proto.webviewResource
 type Build = Proto.webviewBuildRecord
 
+export const triggerUpdate = (name: string): void => {
+  let url = `/api/trigger`
+
```

Comment:
I wonder if it's worth a human-readable error message (even just in the console) if you send a "trigger" command for the tiltfile or for all resources (which ends up being a trigger command for NO resources). (The former is less important cuz I hope to have that implemented sometime next week tho, so 🤷‍♀️ )

Reply:
eh, you can already see this in the console via the network logging